policy_set_integration_test.go

```diff
@@ -154,6 +154,34 @@ func TestPolicySetsCreate(t *testing.T) {
 		assert.Equal(t, ps.Workspaces[0].ID, wTest.ID)
 	})
 
+	t.Run("with policies and projects provided", func(t *testing.T) {
```

Comment:
We are adding workspaces as well so maybe "with policies, workspaces, and projects provided"

Side note: Is there a reason why you are doing both createWorkspace and createProject in this test instead of just createProject?

Reply:
adding workspaces allows to test that both project and workspaces work together as well.